lms/static/scripts/frontend_apps/components/Dialog.js

```diff
@@ -68,48 +67,41 @@ export default function Dialog({
 
   return (
     <div
+      className={classNames({
+        Dialog__background: true,
+        [contentClass]: true,
+      })}
       role="dialog"
       aria-labelledby="Dialog__title"
       onKeyDown={handleKey}
       tabIndex="0"
       ref={rootEl}
     >
-      <div
-        className="Dialog__background"
-        style={{ zIndex: zIndexScale.dialogBackground }}
-      />
```

Comment:
The reason why the dialog background is a separate element is to enable it to be transparent without the content of the dialog inheriting the transparency. MDN [does suggest](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity) an alternative way of achieving this by using the `background` property, which I'd overlooked.

Reply:
^ I've now implemented it using the background property which as far as I can tell is a better way of solving this issue.